samples/storageclass/isilon-replication.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: isilon-replication
+provisioner: csi-isilon.dellemc.com
+reclaimPolicy: Delete
+allowVolumeExpansion: true
+parameters:
+  # replicationPrefix parameter in values.yaml must be used as prefix for all replication parameters in storage class
+  # for e.g., all replication parameters have prefix: replication.storage.dell.com here
+
+  # replication.storage.dell.com/isReplicationEnabled:
+  # Allowed values:
+  #   true: enable replication sidecar
+  #   false: disable replication sidecar
+  # Optional: true
+  # Default value: false
+  replication.storage.dell.com/isReplicationEnabled: "true"
+
+  # replication.storage.dell.com/remoteStorageClassName:
+  # Allowed values: string
+  # Optional: true
+  # Default value: None
+  replication.storage.dell.com/remoteStorageClassName: "isilon-replication"
+
+  # replication.storage.dell.com/remoteClusterID: point to correct remote cluster id
+  # Allowed values: string
+  # Optional: true
+  # Default value: None
+  replication.storage.dell.com/remoteClusterID: "target"
+
+  # replication.storage.dell.com/remoteSystem: point to correct remote PowerScale system, this should match whatever clusterName you specified in `isilon-creds` secret
+  # Allowed values: string
+  # Optional: true
+  # Default value: None
+  replication.storage.dell.com/remoteSystem: "cluster-2"
+
+  # replication.storage.dell.com/rpo: change to any other RPO predefined in a list
+  # Allowed values: "Five_Minutes", "Fifteen_Minutes", "Thirty_Minutes", "One_Hour", "Six_Hours", "Twelve_Hours", "One_Day"
+  # Optional: true
+  # Default value: None
+  replication.storage.dell.com/rpo: Five_Minutes
+
+  # replication.storage.dell.com/ignoreNamespaces: set to 'true' if you want to ignore namespaces and use one volume group
+  # Allowed values:
+  #   true: ignore namespaces and use one volume group
+  #   false: create separate volume group per namespace
+  # Optional: true
+  # Default value: None
+  replication.storage.dell.com/ignoreNamespaces: "false"
+
+  # replication.storage.dell.com/volumeGroupPrefix: volume group prefix
+  # Allowed values: string
+  # Optional: true
+  # Default value: None
+  replication.storage.dell.com/volumeGroupPrefix: "csi"
+  # The name of the access zone a volume can be created in
```

Comment:
Add a new line to segregate the sections.

Reply:
done